<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CategoryDialog_base</class>
 <widget class="QWidget" name="CategoryDialog_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Categories</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="IncidenceEditorNG::AutoCheckTreeWidget" name="mCategories">
     <property name="columnCount">
      <number>1</number>
     </property>
     <column>
      <property name="text">
       <string>Category</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="KLineEdit" name="mLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="mButtonAdd">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mButtonRemove">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KColorCombo" name="mColorCombo"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorCombo</class>
   <extends>QComboBox</extends>
   <header>kcolorcombo.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>IncidenceEditorNG::AutoCheckTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>autochecktreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mCategories</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
